src/__test__/utils/Cluster.ts

```diff
@@ -11,6 +11,7 @@ import type { EndPoint } from "../../types";
 
 import { testDebug } from "./debug";
 import { dockerImages } from "./dockerImages";
+import { delay } from "./delay";
```

Comment:
import not used

Reply:
We should probably get ts to check for this, it's either:

this:
https://www.typescriptlang.org/tsconfig#verbatimModuleSyntax

or maybe:
https://www.typescriptlang.org/tsconfig#noUnusedLocals

Reply:
(not in this PR)